scipy/stats/tests/test_survival.py

```diff
@@ -49,3 +81,63 @@ def test_nonunique(self):
         assert_equal(res.x, ref_x)
         assert_equal(res.cdf, ref_cdf)
         assert_equal(res.sf, ref_sf)
+
+    # ref. [1] page 91
+    t1 = [37, 43, 47, 56, 60, 62, 71, 77, 80, 81]  # times
+    d1 = [0, 0, 1, 1, 0, 0, 0, 1, 1, 1]  # deaths (not censored)
```

Comment:
I understand that the intent is to specify what 1 and 0 mean. In that case, this would be correct:
```suggestion
    d1 = [0, 0, 1, 1, 0, 0, 0, 1, 1, 1]  # 1 means death (i.e. not censored)
```
Only if an observation is not censored do we know when the death occurred. If it is censored, we only have a minimum value of the survival time; death occurred after the observation period.